src/app/utils/codemirror/eslint-lint.js

```diff
@@ -354,7 +354,15 @@ export function fix(source: string) {
 }
 
 export default (async function initialize() {
-  if (!window.eslint && linter === null) {
+  // ugly hack to work around eslint dependency which redefines the require instance on window
+  // this breaks the monaco loader.js
+  // the following workaround can be removed after migration to monaco (vim mode) is done
+  const origRequire = window.require;
```

Comment:
Thats the hacky part of the change. Because eslint will override the window.require reference. So I store the original reference, load eslint and afterwards restore the original reference.